packages/twenty-front/src/modules/ui/layout/table/components/TableSection.tsx

```diff
@@ -30,7 +30,8 @@ const StyledSectionHeader = styled.div<{ isExpanded: boolean }>`
 const StyledSection = styled.div<{ isExpanded: boolean }>`
   max-height: ${({ isExpanded }) => (isExpanded ? '1000px' : 0)};
```

Comment:
I agree with grep, lets keep overflow as it is !
```suggestion
   max-height: ${({ isExpanded }) => (isExpanded ? 'fit-content' : 0)};
```